collector/src/test/java/org/apache/hertzbeat/collector/collect/common/cache/CommonCacheTest.java

```diff
@@ -17,40 +17,91 @@
 
 package org.apache.hertzbeat.collector.collect.common.cache;
 
-import org.junit.jupiter.api.AfterEach;
+import java.util.Optional;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import static org.junit.jupiter.api.Assertions.*;
 
 /**
  * Test case for {@link ConnectionCommonCache}
  */
+@ExtendWith(MockitoExtension.class)
 class CommonCacheTest {
 
+    @Mock
+    private AbstractConnection<?> mockConnection;
```

Comment:
`ConnectionCommonCache` does not depend on external services, so there is no need to use `mock`.

Here you can create an anonymous `AbstractConnection` class for testing.

Reply:
Thanks, PTAL.